i18n/de/fotema.ftl

```diff
@@ -0,0 +1,246 @@
+# SPDX-FileCopyrightText: © 2024 David Bliss
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+## Terms
+
+# See https://projectfluent.org/fluent/guide/terms.html
+
+-app-name = Fotema
+
+## Main Navigation Pages
+
+# Title for library page, which contains the "all", "months", and "years" pages.
+library-page = Bibliothek
+
+# Title for years album.
+years-album = Jahr
+
+# Title for months album.
+months-album = Monat
+
+# Title for all photos/videos album.
+all-album = Tag
+
+# Title for video album.
+videos-album = Videos
+
+# Title for album of selfies.
+selfies-album = Selfies
+
+# Title for album of iOS live photos and Android motion photos.
+animated-album = Animiert
+
+# Title for album showing all folders.
+folders-album = Ordner
+
+# Title for album showing contents of one folder.
+folder-album = Ordner
+
+## Thumbnail decorations
+
+# Label on month album thumbnails.
+# Variables:
+#   $month - month number (1 through 12).
+#   $year - year e.g., 2024
+# Translator note: do not values in square brackets, such as '[other]'.
+month-thumbnail-label = { $month ->
+   [1] Januar {$year}
+   [2] Februar {$year}
+   [3] März {$year}
+   [4] April {$year}
+   [5] Mai {$year}
+   [6] Juni {$year}
+   [7] Juli {$year}
+   [8] August {$year}
+   [9] September {$year}
+   [10] Oktober {$year}
+   [11] November {$year}
+   [12] Dezember {$year}
+  *[other] {$year}
+}
+
+## About Dialog
+
+# Section header for open source projects acknowledgements.
+about-opensource = Quelloffene Projekte
+
+# Translator note: add one translator per-line to get a translation
+# credit in the Fotema's "About" page.
+about-translator-credits =
+  David Bliss <hello@fotema.app>
```

Comment:
@CodingTurtle2 Do you want to put your name and details here for a translating credit in the Fotema's "about" page?